docs/source/_static/custom.js

```diff
@@ -35,6 +35,7 @@ document.addEventListener('DOMContentLoaded', () => {
         { selector: '.toctree-l1 > a', text: 'Authentication and RBAC' },
         { selector: '.toctree-l1 > a', text: 'Volumes' },
         { selector: '.toctree-l2 > a', text: 'Upgrading API Server' },
+        { selector: '.toctree-l1 > a', text: 'Syncing Code and Artifacts' },
```

Comment:
Actually, I'd propose keeping this out of NEW since we don't want users to think we have this capability only now. 

- How about renaming the title to `Syncing Code, Git, and Files`?

- Also, can we remove L30?